src/main.js

```diff
@@ -24,18 +21,24 @@ var currentTitle = document.querySelector('.cover-title');
 var currentImage = document.querySelector('.cover-image');
 var currentTagLine1 = document.querySelector('.tagline-1');
 var currentTagLine2 = document.querySelector('.tagline-2');
-
+var userCover = document.getElementById('cover');
+var userTitle = document.getElementById('title');
+var userDescriptor1 = document.getElementById('descriptor1');
+var userDescriptor2 = document.getElementById('descriptor2');
+var createBookButton = document.querySelector('.create-new-book-button');
 
 
 window.addEventListener("click", clickHandler);
 window.addEventListener("load", randomCover);
 
 
-//redo for hide element
-// function hideElement(element) {
-//   document.querySelector(`.${element}`).classList.add("hidden");
-// }
+ function showElement(elementVar) {
+  elementVar.classList.remove("hidden")
+}
 
```

Comment:
Great inclusion of these functions. It condenses the code down to make more readable.